doc/index.rst

```diff
@@ -251,6 +251,29 @@ So either `ABICHECK_URL` or `ABICHECK_FALLBACK` must be set.
 The url should point to an archive (\*.tar.\*,\*.zip, \*.tgz or \*.tbz2). The file's basename will be displayed as version.
 As an alternative `ABICHECK_VERSION` can be provided explicitly.
 
+Example ABI checking
+++++++++++++++++++++
+
+Let's look at a few example configs in `.travis.yml in this repository, industrial_ci <https://github.com/ros-industrial/industrial_ci/blob/master/.travis.yml>`_. Note that all of these examples check the external repositories (i.e. not industrial_ci) specified by `_EXTERNAL_REPO`.
+
+Case-1: Check ABI compatibility between current and the specific version.
+
+```
+- ROS_DISTRO=indigo _EXTERNAL_REPO='ros/actionlib#38ce66e2ae2ec9c19cf12ab22d57a8134a9285be' ROS_REPO=ros ABICHECK=true
+```
+
+(TODO: Which branch and commit is going to be used for the check in this case?)
```

Comment:
The branch that 'industrial_ci` is run for (e.g. `kinetic-devel` of `ros_canopen`) gets tested against the `0.7.5` tag.